common-lib/src/cmr/common/concepts.clj

```diff
@@ -3,21 +3,18 @@
   (:require [clojure.set :as cset]
             [clojure.string :as str]
             [cmr.common.util :as util]
+            [cmr.common.generics :as common-generic]
             [cmr.common.services.errors :as errors]))
 
-(def generic-concept-prefix->concept-type
-  "Maps a generic concept id prefix to the concept type."
-  {"X" :generic
-   ;; TODO: Generic work: Can I read a non common config file to read in the generics instead of coding them? This is primarily used for validation.
-   "DQS" :dataqualitysummary
-   "OO" :orderoption
-   "SO" :serviceoption
-   "SE" :serviceentry
-   "GRD" :grid})
-
 (def generic-concept-types->concept-prefix
-  "Gets an array of generic concept types."
-  (cset/map-invert generic-concept-prefix->concept-type))
+  "Gets an array of generic concept types.
+   Return {:generic (str X) :grid (str GRD)...}"
+  (merge {:generic "X"} (common-generic/approved-generic-concept-prefixes)))
+
```

Comment:
bamboo thinks I have a syntax error in generic-concept-types->concept-prefix, does anyone see it?

Reply:
solved.